demos/cache_sidechannel.cc

```diff
@@ -115,7 +115,7 @@ std::pair<bool, char> CacheSideChannel::RecomputeScores(
 
 std::pair<bool, char> CacheSideChannel::AddHitAndRecomputeScores() {
   static size_t additional_offset_counter = 0;
-  size_t mixed_i = ((additional_offset_counter * 167) + 13) & 0xFF;
+  size_t mixed_i = ((additional_offset_counter * 167) + 13) & 0x7F;
```

Comment:
I think we should avoid the assumption that we will only ever leak ASCII text.

Reply:
That was a mistake. Reverted.